Public/Format-DockerImageName.ps1

```diff
@@ -0,0 +1,27 @@
+function Format-DockerImageName {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$true)]
+        [string]
```

Comment:
If you would like the validation to happen earlier, consider adding this validation part, `[ValidateScript({Test-Path $_ })]`. But this prevents you from throwing [`FileNotFoundException`](https://github.com/3shapeAS/dockerbuild-pwsh/pull/13/files#diff-f969702d0fbc2de0d5d5475995986148R12).